src/lib/bakeryjs/eval/every.ts

```diff
@@ -0,0 +1,9 @@
+export default function every<T>(arr: T[], evalFunction: (val: T) => boolean) {
```

Comment:
I don't like much the idea of recreating the native Array functions, but this one looks ok.  
The Array.every polyfill looks pretty ugly indeed: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every#polyfill

I just wonder if the processing saved is worth it, but i'm ok with it.

Reply:
We'll see. Based on the simplicity of our use-case and the performance gain at scale illustrated by the attached photos from JSBench, I think it's worth it.